kratos/tests/cpp_tests/sources/test_table.cpp

```diff
@@ -5,10 +5,10 @@
 //                   Multi-Physics
 //
 //  License:         BSD License
-//                     Kratos default license: kratos/license.txt
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Vicente Mataix Ferrandiz
-//
+//                   Anne van de Graaf
```

Comment:
```suggestion
```

[FYI](https://github.com/KratosMultiphysics/Kratos/wiki/Authorship-of-Kratos-files)